ChangeLog

```diff
@@ -1,3 +1,7 @@
+Version 1.7.1
+-------------
+    * Updated xmlrunner package
```

Comment:
Perhaps it's also worth mentioning #158 here. The other commits since v1.7.0 are either internal changes or are to do with the wheel resolver, which isn't distributed as a tool yet, so we can ignore those.

Reply:
Done!